test/write.js

```diff
@@ -114,3 +114,18 @@ test('hex of write{Uint,Int}{8,16,32}{LE,BE} with overflow', function (t) {
   }
   t.end()
 })
+test('android issue', function (t) {
```

Comment:
I'm still sussing out the root issue will totally give it a decent test name once it's all ready